Mage.Sets/src/mage/cards/k/KinzuOfTheBleakCoven.java

```diff
@@ -0,0 +1,156 @@
+package mage.cards.k;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.DiesCreatureTriggeredAbility;
+import mage.abilities.costs.Cost;
+import mage.abilities.costs.CostImpl;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.CreateTokenCopyTargetEffect;
+import mage.abilities.effects.common.DoIfCostPaid;
+import mage.abilities.keyword.FlyingAbility;
+import mage.abilities.dynamicvalue.common.StaticValue;
+import mage.abilities.keyword.ToxicAbility;
+import mage.cards.Card;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.filter.FilterPermanent;
+import mage.filter.common.FilterControlledCreaturePermanent;
+import mage.filter.predicate.mageobject.AnotherPredicate;
+import mage.filter.predicate.permanent.TokenPredicate;
+import mage.game.Game;
+import mage.game.permanent.PermanentCard;
+import mage.players.Player;
+import mage.util.CardUtil;
+import java.util.UUID;
+
+/**
+ * @author miesma
+ */
+public final class KinzuOfTheBleakCoven extends CardImpl {
+
+    private static final FilterPermanent filter
+            = new FilterControlledCreaturePermanent("another nontoken creature you control");
+
+    static {
+        filter.add(AnotherPredicate.instance);
+        filter.add(TokenPredicate.FALSE);
+    }
+
+    public KinzuOfTheBleakCoven(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{4}{B}");
+
+        this.addSuperType(SuperType.LEGENDARY);
+        this.subtype.add(SubType.PHYREXIAN);
+        this.subtype.add(SubType.VAMPIRE);
+        this.power = new MageInt(5);
+        this.toughness = new MageInt(4);
+
+        // Flying
+        this.addAbility(FlyingAbility.getInstance());
+
+        // Whenever another nontoken creature you control dies, you may pay 2 Life and exile it. If you do, create a token that's a copy of that card, except it’s 1/1 and has toxic 1.
+        Effect effect = new DoIfCostPaid(new mage.cards.k.KinzuOfTheBleakCovenEffect(), new KinzuOfTheBleakCovenCost(2), "Pay 2 Life and exile it?");
+        this.addAbility(new DiesCreatureTriggeredAbility(effect,false, filter, true));
+
+    }
+
+    private KinzuOfTheBleakCoven(final mage.cards.k.KinzuOfTheBleakCoven card) {
+        super(card);
+    }
+
+    @Override
+    public mage.cards.k.KinzuOfTheBleakCoven copy() {
+        return new mage.cards.k.KinzuOfTheBleakCoven(this);
+    }
+}
+
+class KinzuOfTheBleakCovenEffect extends OneShotEffect {
+
+    KinzuOfTheBleakCovenEffect() {
+        super(Outcome.Benefit);
+        staticText = "create a token that’s a copy of that creature" +
```

Comment:
The ’ characters here is what's breaking the build. Switching to ' will fix :)

Reply:
Thank you very much :)